### DISCUSSION TITLE
Linear vu-meter missing from latest version ?

### DISCUSSION THREAD

**Discussion by AurelFR** [audacious-media-player/audacious#1477] [General]
I just try to find a vu-meter (linear one )and i cannot find it anymore .We have in plugins only 3 options .

**Comment by radioactiveman** [comment]
The VU-Meter plugin is only available when using Qt. You are probably using GTK, check the configuration at `Settings -> Appearance`.

**Comment by AurelFR** [comment]
Thank you !Working nice .
